.opensource/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Firebase Admin SDK - .NET",
+  "platforms": [
+      ".NET",
```

Comment:
I've got a feeling this will cause a parse error ... how about `DotNet` or `CSharp`?  Something alphanumeric

Reply:
Done